contents/blog/product-health-metrics.md

```diff
@@ -0,0 +1,127 @@
+
+![New user growth](../images/blog/product-health-metrics/growth.png)
+
+### Churn rate
+
+[Churn rate](/blog/customer-churn-analysis-guide) is the rate users stop using the product. When a user churns, it means they haven’t returned to use the product recently. As a product health metric, churn lets you know whether users are leaving your product. 
+
+Churn rate is a key metric for many products and businesses. For SaaS or products relying on recurring usage (and revenue), churn might be the most important metric. This is because it is one of the key drivers of revenue and growth (along with new user growth).
+
+![Churn rate](../images/blog/product-health-metrics/churn.png)
+
+Some products have a naturally high churn, so having a high churn rate isn’t always a bad sign. A dramatic increase in churn is a bad sign though. It means something has changed significantly with your product and should inspire an intervention.
+
+### Daily, weekly, and monthly active users
+
+Between new user growth and churn rate is active users. This is a measure of users using your product calculated on a daily, weekly, or monthly basis.
+
+Consistent growth in active users means:
+
+- signups are continuing
+- you are keeping users engaged
+- users aren’t leaving
+
+Active users are the lifeblood of your product, so seeing them continue to grow means a product is healthy. Blood must continue to flow for your body to work, the same is true with your product.
+
+Doing a ratio of these values such as DAU/MAU can also be useful. This helps you understand how frequently people are engaging with the product. You can use PostHog’s formula mode to calculate this. 
+
+![DAU/MAU ratio](../images/blog/product-health-metrics/daumau.png)
+
+### Feature usage
+
+Having users is great, but you want them to actually use your product. Measuring feature usage helps you do this. This is tracking the usage of parts of your product, such as creating, analyzing, or sharing content.
+
+Feature usage helps you understand the health of individual parts of your product. Like the body, if you heal the "sick" parts, the whole is healthier. Feature usage improves prioritization by aligning development time with usage.
+
+Feature usage is a more specific health metric than new user growth or churn rate. It shows the health of specific features, which enables it to be more actionable.
+
+### Session duration
+
+Depending on the product, session duration, or its average, are useful product health metrics. It is a general measurement of user engagement. This is because it shows if users are spending significant time in your product. If this was to change significantly, something fundamentally shifted in your app.
+
+For some products, the extremes of session duration might be more important. This enables you to monitor the health of your product for power users, which could be different, but more important, than average users.
```

Comment:
For some products? Let's add some specific examples here – also worth adding when session duration isn't useful / which direction matters? i.e. for some products, a reduction might be a success metric?

Reply:
Made two more specific examples, power users (extreme) and utilities (go down)